python/src/TruncatedDistribution.i

```diff
@@ -6,5 +6,7 @@
 
 %include TruncatedDistribution_doc.i
 
+%ignore OT::TruncatedDistribution::getSimplifiedVersion;
```

Comment:
Why do you need to hide this method? If I remember well, Anne was using it in her studies some time ago.